scripts/provisioning-terraform.sh

```diff
@@ -168,6 +168,17 @@ TERRAFORM_TFVARS_PATH="${TERRAFORM_ENVIRONMENT_DIR}/terraform.tfvars"
 
 # "${HOME}"/.config/gcloud/application_default_credentials.json is a well-known location for application-default credentials
 GOOGLE_APPLICATION_CREDENTIALS_PATH="/root/.config/gcloud/application_default_credentials.json"
+RUNTIME_SCRIPT_FOLDER="$(mktemp -d)"
+
+cat <<EOF >"${RUNTIME_SCRIPT_FOLDER}/run.sh"
+if ! gsutil list -p "${DEFAULT_PROJECT}" | grep "gs://tf-state-${DEFAULT_PROJECT}/" ; then
```

Comment:
Put `tf-state-${DEFAULT_PROJECT}` in a variable and use it here and when you init the backend so that you don't define the same thing in two different places.

Reply:
updated